apollo-router/Cargo.toml

```diff
@@ -25,7 +25,7 @@ otlp-http = ["opentelemetry-otlp/http-proto"]
 
 [dependencies]
 anyhow = "1.0.55"
-apollo-parser = { git = "https://github.com/apollographql/apollo-rs.git", rev = "9a14371f3e5352920c74dcae85fa4e71b24f0449" }
+apollo-parser = { git = "https://github.com/apollographql/apollo-rs.git", rev = "e707e0f78f41ace1c3ecfe69bc10f4144ffbf7ac" }
```

Comment:
could you add the branch name? There's no fundamental difference, the commit id is the most precise reference, it will just be slightly easier to look up for us

Reply:
Hmm, do you mean add a `branch` parameter to the dependency? If I do, then I can't specify the `rev`, since it's an either/or situation for those parameters.
That's probably ok here, since we'll always want the latest `rev` on that `branch`, but I think that will mean that the router will re-build when that branch updates. Not sure if we want that.

Reply:
Oh, I thought we could specify both. It's ok with rev then